src/pyspdlog.cpp

```diff
@@ -403,7 +403,7 @@ PYBIND11_MODULE(spdlog, m) {
 
     m.def("set_async_mode", set_async_mode, 
             py::arg("queue_size") = 1 << 16,
-            py::arg("async_overflow_policy") = AsyncOverflowPolicy::block_retry,
+            py::arg("async_overflow_policy") = 0,    // 0 equals AsyncOverflowPolicy::block_retry - using block_retry results in gcc 5.4 compiler error
```

Comment:
is this backward compatible?

Reply:
Oh I see its equivalent

Reply:
pyspdlog is POSIX compliant

Reply:
We should add jenkins test in the future